Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/OfferViewHolder.java

```diff
@@ -14,13 +14,16 @@
 import com.stxnext.volontulo.api.Offer;
 import com.stxnext.volontulo.ui.utils.BaseViewHolder;
 
+import org.parceler.Parcels;
+
 import butterknife.Bind;
 import butterknife.OnClick;
 
 class OfferViewHolder extends BaseViewHolder<Offer> {
 
     private int id;
     private String imagePath;
+    private Offer offer;
```

Comment:
Please note that `BaseViewHolder` provide protected field `objectBinded` for this functionality.